templates/search/releasecalendar/main.go

```diff
@@ -22,23 +19,22 @@ func main() {
 	var (
 		sr = query.ReleaseSearchRequest{
 			Size:           10,
-			SortBy:         query.RelDateDesc,
+			SortBy:         query.Relevance,
 			Term:           "Education in Wales",
 			ReleasedAfter:  query.MustParseDate("2018-01-01"),
 			ReleasedBefore: query.MustParseDate("2018-12-31"),
 			Type:           query.Published,
-			Highlight:      false,
-			Now:            query.Date(time.Now()),
+			Provisional:    true,
+			Confirmed:      true,
+			Postponed:      true,
+			Highlight:      true,
 		}
-		builder             *query.ReleaseBuilder
-		q, uq, responseData []byte
-		err                 error
-		ctx                                         = context.Background()
-		multi                                       = flag.Bool("multi", false, "use the multi query format when sending the query to ES")
-		file                                        = flag.String("file", "", "a file containing the actual query to be sent to ES (in json format)")
-		esClient                                    = elasticsearch.New("http://localhost:9200", dphttp.NewClient(), "eu-west-1", "es")
-		esSearch                                    = esClient.Search
-		esTransformer       api.ResponseTransformer = transformer.NewReleaseTransformer()
+		builder         *query.ReleaseBuilder
+		q, responseData []byte
+		err             error
+		ctx             = context.Background()
+		file            = flag.String("file", "", "a file containing the actual multi-query to be sent to ES (in json format)")
+		esClient        = elasticsearch.New("http://localhost:9200", dphttp.NewClient(), "eu-west-1", "es")
```

Comment:
(minor): i guess this should be changed ```elasticsearch.New("http://localhost:9200", dphttp.NewClient(), "eu-west-1", "es")``` so that the elastic url can be loaded from config, same as aws region and service.